react-carousel-component/my-app/src/Carousel.js

```diff
@@ -0,0 +1,66 @@
+import CircleButton from './CircleButton';
+import ArrowButton from './ArrowButton';
+import CarouselImage from './CarouselImage';
+import './Carousel.css';
+import { useState, useEffect, useCallback } from 'react';
+
+export default function Carousel({ itemsArray }) {
+  const [bannerIndex, setBannerIndex] = useState(0);
+
+  /**
+   * I created an increment callback function so the 'useEffect' looks cleaner. Otherwise,
+   * I would just have the same setBannerIndex call inside useEffect and the dependencies
+   * would be bannerIndex and itemsArray.length, instead of incrementCB. I wasn't sure
+   * which was more 'industry standard'
+   */
+  const incrementCB = useCallback(() => {
+    setBannerIndex((bannerIndex + 1) % itemsArray.length);
+  }, [bannerIndex, itemsArray.length]);
+
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      incrementCB();
+    }, 3000);
+    return () => clearTimeout(timerId);
+  }, [incrementCB]);
+
+  function increment() {
+    // used with the Right Arrow
+    setBannerIndex((bannerIndex + 1) % itemsArray.length);
```

Comment:
This is the same as `incrementCB`. Why not just delete this and always use `incrementCB`? (You can use a `useCallback` function exactly like you would a normal function.)